app/models/manageiq/providers/azure/cloud_manager/vm.rb

```diff
@@ -1,7 +1,9 @@
 class ManageIQ::Providers::Azure::CloudManager::Vm < ManageIQ::Providers::CloudManager::Vm
+  include_concern 'Operations'
+
   def provider_object(connection = nil)
     connection ||= ext_management_system.connect
-    connection.instances[ems_ref]
+    connection
```

Comment:
The VirtualMachineService object isn't the actual VM. It's basically a wrapper for getting VM information from Azure. I can't call start without the name and group, and I don't see a way to get that using with_provider_object.